docs/UserGuide.md

```diff
@@ -343,7 +350,7 @@ ___
 
 ### Finding an anime: `find`
 
-Finds all anime(s) that match specified keyword(s) in their names or genres
+Finds all anime(s) in the current tab that match specified keyword(s) in their names or genres
```

Comment:
Um i think we need to come up with a better phrase for this, cos if the user uses `find` twice it doesnt find from the current tab but instead from the list with specific `status` / `all`. Need help to rephrase this.

Reply:
maybe use "with status corresponding to the current tab"

Reply:
changed to this, but may be confusing if user is in all tab